kb-search/requirements.txt

```diff
@@ -1,3 +1,3 @@
-nltk==3.4.1
```

Comment:
Could you go ahead and make sure all NLTK version throughout Katecheo are upgraded as prescribed here: https://github.com/cvdigitalai/katecheo/pull/17.